### DISCUSSION TITLE
Welcome to Panda3DS Discussions!

### DISCUSSION THREAD

**Discussion by wheremyfoodat** [wheremyfoodat/Panda3DS#137] [Announcements]
Gang

**Comment by georgemoralis** [comment]
ψηνεται ο κώλος σου...

**Comment by liuk7071** [comment]
Gang

**Reply by wheremyfoodat** [reply]
Gang

**Comment by OFFTKP** [comment]
Gang

**Reply by wheremyfoodat** [reply]
Safe word must be at least 8 characters long and contain 1 special character

**Comment by sakuramipha** [comment]
Gang

**Reply by wheremyfoodat** [reply]
Omg gang

**Reply by liuk7071** [reply]
gang!!!

**Comment by ghost** [comment]
fortnite balls

**Comment by KellanClark** [comment]
ace 🂡 lightning 🌩 2 🕑 wont ❌ save 🦎 on 🔛 gbarunner 2 😳 under 😡 openagbfirm ??????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????

**Reply by wheremyfoodat** [reply]
Maybe if this was written in Rust...

**Comment by caldog20** [comment]
hey fam

**Comment by technicaljicama** [comment]
Will panda have a gles2 renderer in the future?

**Reply by wheremyfoodat** [reply]
No. OpenGL ES 2 is a very old and limited API that lacks a bunch of things needed for 3DS emulation. For comparison it's hard to keep the desktop GL requirements to 4.1 for MacOS support, which is a lot more complete of an API than ES 2.0

**Comment by ghost** [comment]
my heart is...
